asv.conf.json

```diff
@@ -4,6 +4,7 @@
   "project_url": "https://github.com/unitaryfund/mitiq",
   "repo": "https://github.com/unitaryfund/mitiq.git",
   "branches": ["master"],
+  "pythons": ["3.8"],
```

Comment:
I'm also not totally sure how we would test with multiple versions of python on github actions since we specify the python version we want to use in the action. Can probably come up with a workaround, but I don't think it's 100% necessary.

Reply:
On second attempt, it seems github actions does not play well python 3.7. [e076cb2](https://github.com/unitaryfund/mitiq/pull/1354/commits/e076cb2044130faba725f4ae043811f0938f0202) failed due to python 3.7 being missing, even though I tried to setup python 3.7.

I think using 3.8 might be okay as many of our test run only on 3.8, and we are planning to support newer versions of python soon (and hence likely drop support for older versions).

Reply:
SGTM!